src/site/content/en/blog/read-files/index.md

```diff
@@ -4,86 +4,59 @@ subhead: How to select files, read file metadata and content, and monitor read p
 description: |
   How to select files, read file metadata and content, and monitor read progress.
 date: 2010-06-18
-updated: 2020-03-24
+updated: 2020-05-08
 authors:
  - kaycebasques
+ - petelepage
 tags:
   - blog
-draft: true
+  - storage
+hero: hero.jpg
+alt: 'A 3.5" floppy disk'
```

Comment:
```suggestion
```
This looks funky to me so I'm just going to drop it.

![image](https://user-images.githubusercontent.com/4713486/81637067-49429a80-93ca-11ea-81b3-0ab97e724a66.png)

![image](https://user-images.githubusercontent.com/4713486/81637086-5495c600-93ca-11ea-9515-ff125d9e3cb6.png)